plugins/modules/cloud/scaleway/scaleway_database_backup.py

```diff
@@ -0,0 +1,365 @@
+    choices:
+        - present
+        - absent
+        - exported
+        - restored
+
+  region:
+    description:
+        - Scaleway region to use (for example C(fr-par)).
+    type: str
+    required: true
+    choices:
+        - fr-par
+        - nl-ams
+
+  id:
+    description:
+        - UUID used to identify the database backup.
+        - Required for C(absent), C(exported) and C(restored) states.
+    type: str
+
+  name:
+    description:
+        - Name used to identify the database backup.
+        - Required for C(present) state.
+        - Ignored when C(state=absent), C(state=exported) or C(state=restored).
+    type: str
+    required: false
+
+  database_name:
+    description:
+        - Name used to identify the database.
+        - Required for C(present) and C(restored) states.
+        - Ignored when C(state=absent) or C(state=exported).
+    type: str
+    required: false
+
+  instance_id:
+    description:
+        - UUID of the instance associated to the database backup.
+        - Required for C(present) and C(restored) states.
+        - Ignored when C(state=absent) or C(state=exported).
+    type: str
+    required: false
+
+  expires_at:
+    description:
+        - Expiration datetime of the database backup (ISO 8601 format).
+        - Ignored when C(state=absent), C(state=exported) or C(state=restored).
+    type: str
+    required: false
+
+  wait:
+    description:
+        - Wait for the instance to reach its desired state before returning.
+    type: bool
+    default: 'no'
+
+  wait_timeout:
+    description:
+        - Time to wait for the backup to reach the expected state.
+    type: int
+    required: false
+    default: 300
+
+  wait_sleep_time:
+    description:
+        - Time to wait before every attempt to check the state of the backup.
+    type: int
+    required: false
+    default: 3
+'''
+
+EXAMPLES = '''
+  - name: Create a backup
+    community.general.scaleway_database_backup:
+        name: 'my_backup'
+        state: present
+        region: 'fr-par'
+        database_name: 'my-database'
+        instance_id: '50968a80-2909-4e5c-b1af-a2e19860dddb'
+
+  - name: Export a backup
+    community.general.scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: exported
+        region: 'fr-par'
+
+  - name: Restore a backup
+    community.general.scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: restored
+        region: 'fr-par'
+        database_name: 'my-new-database'
+        instance_id: '50968a80-2909-4e5c-b1af-a2e19860dddb'
+
+  - name: Remove a backup
+    community.general.scaleway_database_backup:
+        id: '6ef1125a-037e-494f-a911-6d9c49a51691'
+        state: absent
+        region: 'fr-par'
+'''
+
+RETURN = '''
+data:
+    description: Backup metadata.
+    returned: when C(state=present), C(state=exported) or C(state=restored)
+    type: dict
+    sample: {
+        "data": {
```

Comment:
```suggestion
metadata:
    description: Backup metadata.
    returned: when C(state=present), C(state=exported) or C(state=restored)
    type: dict
    sample: {
        "metadata": {
```
maybe the key `data` => `metadata` or `status` or something else?